README.md

```diff
@@ -0,0 +1,37 @@
+# Künstliche Intelligenz für Mühle
+
+Dieses Repository beinhaltet eine Python Implementierung für eine künstliche Intelligenz, die das Brettspiel Mühle spielen kann.
+Hierbei werden die Algorithmen `Minimax` und `Alpha-Beta-Pruning` verwendet.
+Als Grundlage dient das [Skript (Kapitel 4 *Playing Games*)](https://github.com/karlstroetmann/Artificial-Intelligence) von Prof. Dr. Karl Stroetmann.
+
+![](images/player-vs-alphabetapruning.png)
+
+Da dieses Repository die Arbeit von einer Studienarbeit und zwei Hausarbeiten beinhaltet, werden diese hier genau aufgeschlüsselt:
+
+## Studienarbeit
+
+**Authoren**: @nclskfm, @benfu99 und @jstsddg
```

Comment:
Autoren wird im deutschen ohne H geschrieben ^^ is überall verkehrt, markier ich dir aber jetzt nicht alles :P

Reply:
Danke, ich habe es gefixt. :)